src/clr-angular/utils/drag-and-drop/providers/drag-handle-registrar.spec.ts

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright (c) 2016-2018 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+import {Component} from "@angular/core";
+import {ComponentFixture, TestBed} from "@angular/core/testing";
+import {ClrDragAndDropEventBus} from "./drag-and-drop-event-bus";
+import {ClrDragEventListener} from "./drag-event-listener";
+import {MOCK_DRAG_EVENT_LISTENER_PROVIDER} from "./drag-event-listener.mock";
+import {ClrDragHandleRegistrar} from "./drag-handle-registrar";
+
+export default function(): void {
+    describe("Drag Handle Registrar", function() {
+        let fixture: ComponentFixture<any>;
+        let testComponent: DragHandleTestComponent;
+
+        const draggableEl = document.createElement("div");
+        const customHandleEl = document.createElement("button");
+
+        // Providers
+        let dragHandleRegistrar: any;
+        let dragEventListener: any;
+
+        beforeEach(() => {
+            TestBed.configureTestingModule(
+                {declarations: [DragHandleTestComponent], providers: [ClrDragAndDropEventBus]});
+            fixture = TestBed.createComponent(DragHandleTestComponent);
+            testComponent = fixture.componentInstance;
+            dragHandleRegistrar = fixture.debugElement.injector.get(ClrDragHandleRegistrar);
+            dragEventListener = fixture.debugElement.injector.get(ClrDragEventListener);
+        });
+
+        afterEach(() => {
+            fixture.destroy();
+        });
+
+        it("should be able to register element as default handle on assignment", function() {
```

Comment:
Nitpick: "should be able to" is redundant with the fact that this is a spec. You can even see that by just looking at the other unit tests: every single one starts with "should be able to", because that's the entire point of a test. 😛 

Just `it("registers element as default handle on assignment", ...)`.

Reply:
Updated the messages.